syft/frameworks/torch/hook_args.py

```diff
@@ -209,9 +211,12 @@ def hook_response(attr, response, wrap_type, wrap_args={}, new_self=None):
     if not response_is_tuple:
         response = (response, 1)
 
-    attr_id = "{}@{}.{}".format(attr, wrap_type.__name__, hash(frozenset(wrap_args.items())))
+    hash_wrap_args = hash(frozenset(wrap_args.items()))
+    attr_id = f"{attr}@{wrap_type.__name__}.{response_is_tuple}.{hash_wrap_args}"
 
     try:
+        assert attr not in exclude_functions
```

Comment:
Why are these assertions in a try vs just a Boolean if

Reply:
The try catch not only errors of this assertion but also errors in the two other lines
```
# Load the utility function to transform the args
        response_hook_function = hook_method_response_functions[attr_id]
        # Try running it
        new_response = response_hook_function(response)
```